### ISSUE TITLE
Any botbuilder-timeout module for MS Bot SDK v4 using Node js

### ISSUE THREAD

**Issue by vaatsalya** [User1m/botbuilder-timeout#5]
Hi Team,
I am developing a bot using SDK v4 using node js and trying to implement the 'timeout' functionality using this npm module but i am getting the error saying 'use' is undefined in 'this.bot.use' which might have been changed in SDK v4. Is there any module that works for sdk v4?

**Comment by vpysaran** [comment]
I am also facing same issue.  Azure chat bot SDK v4 with user ideal time 30 sec need to send "are you there?" message.

**Comment by tejasvinishelar** [comment]
Yes I am using the same SDK v4 using nodejs and getting "TypeError: this.bot.use is not a function at Timeout.init" error for timeout.
Is there any update on this?

**Comment by phongnm-0776** [comment]
I am also facing same issue. Please update or release new version of botbuilder-timeout, i really need it. Thanks you so much
